commit f66aac2c7d13d55df9523cd4d2a28374fecaebe2
Author: Trond Myklebust <trond.myklebust@primarydata.com>
Date:   Wed Feb 8 11:29:46 2017 -0500

    NFSv4: Fix memory and state leak in _nfs4_open_and_get_state
    
    commit a974deee477af89411e0f80456bfb344ac433c98 upstream.
    
    If we exit because the file access check failed, we currently
    leak the struct nfs4_state. We need to attach it to the
    open context before returning.
    
    Fixes: 3efb9722475e ("NFSv4: Refactor _nfs4_open_and_get_state..")
    Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
    Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1333_f66aac2c7d13d55df9523cd4d2a28374fecaebe2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1333_f66aac2c7d13d55df9523cd4d2a28374fecaebe2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
